### PR TITLE
Implemented Control+s to save file.

### PR DESCRIPTION
**Pull request by gunjan-l**
## Describe your changes
Changed file [main.py](https://github.com/9gl/PyNote/blob/42-Save_with_control_s/main.py) 
line no 14 and 69
## Checklist before requesting a review
- [x] I have performed a self-review of my code
- [ ] If it is a core feature, I have added thorough tests.
- [ ] Do we need to implement analytics?
- [ ] Will this be part of a product update? If yes, please write one phrase about this update.

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -1,6 +1,7 @@
 import tkinter as tk
 from tkinter import filedialog
 from tkinter import messagebox
+import signal
```
Comment (mrHeavenli) [inline main.py]: Correct me if I'm wrong but I don't think we need that import

**Review by mrHeavenli — CHANGES_REQUESTED** [review]
Looks good to me but afaik we don't need to import signal

**Comment by gunjan-l** [comment]
Yes we don't need that import

File: main.py (hunk 1 above)
Comment (gunjan-l) [reply main.py]: No you are right

**Comment by gunjan-l** [comment]
@mrHeavenli I have made the changes

**Review by mrHeavenli — APPROVED** [review]
Looks good to me

**Comment by mrHeavenli** [comment]
@SamReiCooper or @PatrickBruso  can someone merge this? It's been open for five days now

**Comment by PatrickBruso** [comment]
@9gl @mrHeavenli I'll take a look, thanks for the ping.

**Review by PatrickBruso — APPROVED** [review]
Looks good!  Thanks for the PR.

**Comment by mrHeavenli** [comment]
Thanks

**Pull request merged**